lib/client/index.js

```diff
@@ -52,74 +52,58 @@ module.exports = ({ port = null, config = {}, filters = {} }) => {
       config.brokerClientValidationTimeoutMs || 5000;
     const isJsonResponse = !config.brokerClientValidationJsonDisabled;
 
-    const data = {
-      brokerClientValidationUrl: logger.sanitise(
-        config.brokerClientValidationUrl,
-      ),
-      brokerClientValidationMethod,
-      brokerClientValidationTimeoutMs,
-    };
-
-    const validationRequestHeaders = {
-      'user-agent': 'Snyk Broker client ' + version,
-    };
-
     // set auth header according to config
-    if (config.brokerClientValidationAuthorizationHeader) {
-      validationRequestHeaders.authorization =
-        config.brokerClientValidationAuthorizationHeader;
+    function credsFromHeader(s) {
+      if (s.indexOf(" ") >= 0) {
+        return s.substring(s.indexOf(" ") + 1);
+      } else {
+        return s;
+      }
+    }
+    let auths = [];
```

Comment:
Do you think it makes sense to extract line 64-77 to a function?

Reply:
Yes, I think it does.